README.md

```diff
@@ -310,11 +317,21 @@ glue_client = boto3.client("glue")
 glue_client.create_table(**boto_dict) # Would deploy glue schema based on our metadata
 ```
 
-included alongside `GlueConverter` is `GlueTable` that can overlay a metadata object, dictionary, or path to metadata file. it has one method:
+Included alongside `GlueConverter` is `GlueTable` that can overlay a metadata object, dictionary, or path to metadata file. it has two methods:
 - **generate_from_meta:** generates a glue table from the metadata object, dict, or string path, takes the following arguments:
     - _metadata:_ the metadata object, dict, or string path that is to be overlaid
     - _table\_location:_ a kwarg, the location of the table data. This can also be a property of the metadata object, dict, or file
     - _database\_name:_ a kwarg, the name of the glue database to put the table. This can also be a property of the metadata object, dict, or file
+    
+    If primary_key or/and glue_table_properties are populated in the table schema/metadata, _generate\_from\_meta_ will update the table parameters in the
```

Comment:
I think there is a subtle difference for each since in GlueConverter, it will update the dictionary which is then passed to boto client whereas in GlueTable the method directly updates the table in Glue Data Catalog. I've tried to make this difference clear in the updated documentation.